examples/supervised_contrastive_replay.py

```diff
@@ -0,0 +1,71 @@
+import torchvision.transforms
```

Comment:
I would remove the example. I don't think it adds anything. We will have a script on continual-learning-baselines

Reply:
You can do whatever you want in your local branch, as long as you don't push it ;)

> In case, we should remove all examples like this one, because we have a lot right now that just show how to use a certain strategy.

I agree that we should remove them. You can remove all the examples that are trivial and already available in continual-learning-baselines.